setup.py

```diff
@@ -14,7 +14,7 @@ def wrapper(name, enc=ascii):
         return {True: enc}.get(name == 'mbcs')
     codecs.register(wrapper)
 
-install_requires = ['pytz', 'httplib2', 'beautifulsoup4', 'lxml']
+install_requires = ['pytz', 'beautifulsoup4', 'lxml', 'requests']
```

Comment:
Removed  requirement for httplib2 and instead use requests library for a more modern feel

Reply:
@MLDERES the hero everyone needs.....